docs/external/vfile-plugin.md

```diff
@@ -26,6 +26,57 @@ Detailed documentation can be found on our [GitHub Documentation Page](http://vm
 * All docker swarm managers should open [official etcd ports](http://www.iana.org/assignments/service-names-port-numbers/service-names-port-numbers.txt) 2379 and 2380, or the user-defined ETCD ports (please find more details about user-defined ETCD ports in the Installation section)
 
 ## Installation
+vFile requires all hosts running in Swarm mode, you can use this [script] (https://github.com/vmware/docker-volume-vsphere/tree/master/misc/scripts/vfile-swarm-setup.sh) to setup all hosts in Swarm mode. This script takes a configuration file as parameter. The configuration file has the following format:
+```
+Number of all nodes
+Number of Swarm manager nodes
+IP address of node 1
+IP address of node 2
+...
+IP address of node n
+```
+
+A sample configuration file can be found [here] (https://github.com/vmware/docker-volume-vsphere/tree/master/misc/scripts/vfile-swarm-config.txt)
+
+You need to setup SSH keys for `root` user on all hosts. Then run the following command to configure a Swarm cluster:
+
+```
+~$ ./vfile-swarm-setup.sh vfile-swarm-config.txt
```

Comment:
IMO, only command is sufficient to specify no need of entire output in the user facing document.

Reply:
Done.